### ISSUE TITLE
Transitions API not working on iOS

### ISSUE THREAD

**Issue by Sin-** [software-mansion/react-native-reanimated#884]
## Description
I was just trying a simple fade in transition with the Transition API, and the transition works perfectly on Android device, but no animation on iOS simulator or device. The elements just appear abruptly.

## Steps To Reproduce

1. Init a new react native project with `react-native init`
2. Install react-native-reanimated, then `cd ios && pod install`
3. Replace the App.js code by this:
```
import React, {useEffect, useRef, useState} from 'react';
import {SafeAreaView, Image, ImageBackground, View} from 'react-native';
import {Transitioning, Transition} from 'react-native-reanimated';

const SplashScreen = () => {
  const transition = (
    <Transition.In type="fade" durationMs={1000} interpolation="easeInOut" />
  );
  const ref = useRef();
  let [showBackground, setShowBackGround] = useState(false);
  let [showLogo, setShowLogo] = useState(false);

  useEffect(() => {
    setTimeout(() => {
      ref?.current?.animateNextTransition();
      setShowBackGround(true);
      setTimeout(() => {
        ref?.current?.animateNextTransition();
        setShowLogo(true);
      }, 2000);
    }, 1000);
  });

  return (
    <View style={{flex: 1, backgroundColor: 'black'}}>
      <Transitioning.View transition={transition} ref={ref} style={{flex: 1}}>
        {showBackground && (
          <ImageBackground
            style={{flex: 1, alignItems: 'center'}}
            source={require('./splashScreen.jpg')}>
            <SafeAreaView>
              {showLogo && (
                <Image
                  style={{
                    height: 70,
                    position: 'absolute',
                    top: 100,
                    alignSelf: 'center',
                  }}
                  source={require('./logo.png')}
                  resizeMode="contain"
                />
              )}
            </SafeAreaView>
          </ImageBackground>
        )}
      </Transitioning.View>
    </View>
  );
};

export default SplashScreen;

```
4. Use any image for the background and the logo and run `yarn ios`

### Expected behavior
The background image and the logo should fade in on iOS

### Actual behavior
The elements just appear without animation

## Snack or minimal code example
https://snack.expo.io/uy43ZgQGO

## Package versions

- React: 16.11.0
- React Native: 0.62.2
- React Native Reanimated: 1.9.0

**Comment by Evan-Younan** [comment]
Can confirm that I just ran across this in a similar application. I had copied code from a tutorial which was using reanimated for similar animations and they were working fine then.

The version that was working when I was doing the tutorial was: `~1.4.0`

**Comment by jakub-gonet** [comment]
We discussed Transitions internally in our team. Considering that v2 makes creating animations easier and Transitions was an experimental API for a long time, we decided to leave Transitions as they are.

We won’t actively fix bugs in them, but we’ll happily merge any PR with fixes. We’re also thinking about rewriting Transitions in v2 sometime in the future.

I’m closing this issue, but if you have further questions, I will be happy to help.
